### ISSUE TITLE
netusage@30yavash.com not displaying

### ISSUE THREAD

**Issue by JulianGro** [linuxmint/cinnamon-spices-desklets#945]
### Desklet version/Build date

1.0.5

### Cinnamon version

5.8.4

### Distribution

Debian Testing

### Graphics hardware *and* driver used

Intel A770 16GB with Mesa 23.2.1-1

### Desklet name and maintainer

netusage@30yavash.com @pdcurtis

### What happened?

The netusage desklet doesn't appear. There is an error in .xsession-errors:
```
(cinnamon:2581): Gjs-CRITICAL **: 08:40:40.108: JS ERROR: TypeError: Clutter.Texture is not a constructor
_updateGraph/<@/home/juliangro/.local/share/cinnamon/desklets/netusage@30yavash.com/4.0/desklet.js:144:29
spawnCommandLineAsync/<@/usr/share/cinnamon/js/misc/util.js:206:17
```

### Other information

The error message prompting you to install vnstati works correctly.

**Comment by ChrisDAT20** [comment]
It seems that the desklet is using an deprecated API: Clutter.Texture is not avalable anymore. And as whole Clutter is already unmaintained, most likely thats a deadend :)

**Comment by ChrisDAT20** [comment]
What needs to be done to get the last version with commit 576e031 on Debian, when downloading with the Desklet tool?

**Comment by fredcw** [comment]
In System Settings>Desklets>Download tab, this version should be available. Refresh the list if necessary and an upgrade button for the desklet should appear.

**Comment by ChrisDAT20** [comment]
I had the desklet deinstalled before, refreshed the cache with the Refresh button, then downloaded it again. I checked in .local/share/connamon/desklets, and the version with the fix is there. But when I want to add it with the plus sign, the screen flashes once and the log still states:
JS ERROR: TypeError: Clutter.Texture is not a constructor
I then issues Alt-F2 and opened "lg", and under Extensions I saw Network usage monitor and issued "Reload code". After that, it is now running. But, the old version keeps reporting to the logfile that Clutter.Texture is not known.
I think that this will stop when I restart the desktop?
